anonlinkclient/cli.py

```diff
@@ -641,6 +643,45 @@ def validate_schema(schema):
         raise SystemExit(-1)
 
 
+@cli.command('compare', short_help="compare two schemas")
+@click.argument('schema1', type=click.File('r', lazy=True))
+@click.argument('schema2', type=click.File('r', lazy=True))
+@click.option('-n', 'output_type', flag_value='n', default=True, help='Produce a ndiff format diff (default)')
+@click.option('-u', 'output_type', flag_value='u', help='Produce a unified format diff')
+@click.option('-m', 'output_type', flag_value='m', help='Produce HTML side by side diff')
```

Comment:
I am expecting to produce a file and visualise it a Github style comparison. The stdout is quite confusing with those HTML tags and it is hard to find out what's changed...